lib/workload/orcabus-stateless-stack.ts

```diff
@@ -88,17 +94,13 @@ export class OrcaBusStatelessStack extends cdk.Stack {
         this
       )
     );
-    const databaseSecret = Secret.fromSecretNameV2(
-      this,
-      'FilemanagerDatabaseSecret',
-      config.databaseSecretName
-    );
 
     return new Filemanager(this, 'Filemanager', {
       buckets: config.eventSourceBuckets,
       buildEnvironment: {},
-      databaseSecret,
-      databaseSecurityGroup: this.lambdaSecurityGroup,
+      host: this.rdsMasterSecret.secretValueFromJson('host').unsafeUnwrap(),
```

Comment:
As an alternative, the `host` could be defined in an SSM parameter, instead of using the master secret property.

Reply:
Agree! Perhaps we could have SSM to store the `port` and `host` on the stack where the db is created.

Reply:
The port is configurable from the top-level `constants.ts`, although annoyingly the endpoint url isn't! I can add some SSM parameters which probably make more sense than the master rds secret.